### PR TITLE
DE39772 - Bump my-courses version

### PR DESCRIPTION
**Pull request by svanherk**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1102,9 +1092,9 @@
       }
     },
     "@brightspace-ui-labs/grade-result": {
-      "version": "1.0.15",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/grade-result/-/grade-result-1.0.15.tgz",
-      "integrity": "sha512-tBunojS3yKtZd19G9zEr2nmGK+3ftVH0k8Kd5exOgP3h+j8Sc+ty/5SazNyej/apRtUKRvV4qhh/wpbqDAObpw==",
+      "version": "1.0.16",
```
Comment (svanherk) [inline package-lock.json:1095]: FYI @zhangium

File: package-lock.json [hunk 2]
```diff
@@ -1157,9 +1147,9 @@
       }
     },
     "@brightspace-ui/core": {
-      "version": "1.57.4",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.57.4.tgz",
-      "integrity": "sha512-Q7ZeKaHXzcu5RpPFWgGTAbjof03zW9jLyHtxs+7KxxfVF5h/3QRJJQQEI6vkG5aWZsDxpNlyt8QRMP4f/QPhDw==",
+      "version": "1.58.0",
```
Comment (svanherk) [inline package-lock.json:1150]: FYI @AllanKerr @dbatiste @zinabat @ciss1995

File: package-lock.json [hunk 3]
```diff
@@ -1186,7 +1176,7 @@
       "dev": true
     },
     "@d2l/audio": {
-      "version": "github:Brightspace/d2l-audio#d566da4e1ecf530314b93c6cf0f97a40c47f1efd",
+      "version": "github:Brightspace/d2l-audio#a4c79a3da9e4e6e80764a4dd4ed31acaecd31ce3",
```
Comment (svanherk) [inline package-lock.json:1179]: FYI @jameswklassen

File: package-lock.json [hunk 4]
```diff
@@ -4632,7 +4616,7 @@
       }
     },
     "d2l-cpd": {
-      "version": "github:Brightspace/continuous-professional-development#4f569c29a34388e0e59c9fd677a023fc6d3ecacf",
+      "version": "github:Brightspace/continuous-professional-development#27e57b29d8b65d5cdeb25704a0667712c8f6d142",
```
Comment (svanherk) [inline package-lock.json:4619]: FYI @BraydonKains

File: package-lock.json [hunk 5]
```diff
@@ -4709,7 +4693,7 @@
       }
     },
     "d2l-enrollments": {
-      "version": "github:BrightspaceHypermediaComponents/enrollments#6373a4e65f1d7a2d7091c8664c900a3c864260fc",
+      "version": "github:BrightspaceHypermediaComponents/enrollments#091757d3abe151d031f944a3a417b30c6bde0524",
```
Comment (svanherk) [inline package-lock.json:4696]: FYi @ciss1995

File: package-lock.json [hunk 6]
```diff
@@ -5038,7 +5022,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#b4cdb75e90ada5be0d47caa78f0d94c58a5be76b",
+      "version": "github:BrightspaceHypermediaComponents/organizations#c8b36c7fba36c7988c0016b67b14408b63e8a829",
```
Comment (svanherk) [inline package-lock.json:5025]: FYi @ciss1995

File: package-lock.json [hunk 7]
```diff
@@ -26091,7 +26057,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#77df7fdefe3668330b1150ac4f523e9795664160",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#407a9d47d79845596d6e7254c6a1fa8915fa7242",
```
Comment (svanherk) [inline package-lock.json:26060]: FYI @vix633

**Comment by svanherk** [comment]
@dbatiste @ciss1995 I know you two were coordinating a list update across multiple repos.  Is this all good to go in?

File: package-lock.json (hunk 6 above)
Comment (ciss1995) [reply package-lock.json:5025]: Thanks :)

**Pull request merged**